### ISSUE TITLE
Should be using prepare rather than postinstall step for website dependency install

### ISSUE THREAD

**Issue by benjamind** [mobxjs/mobx#2165]
In the latest Mobx 5.14.1, Installing mobx as a dependency is resulting in the `postinstall` script executing which is trying to install the dependencies of the website folder which is not distributed in the package resulting in errors like this during dependency installation:

```bash
$ yarn --cwd website install
Killed
yarn install v1.16.0
error Directory "/home/circleci/project/node_modules/mobx/website" doesn't exist
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed with exit code 1.
```

Reproduction:
* install as a dependency of any project.

Easy fix is to change `postinstall` to `prepare`. `prepare` will execute only on local installation, and not during installation as a dependency.

**Comment by FredyC** [comment]
Thanks for the report, that's definitely bad and ugly bug. We will sort it out soon.

**Comment by FredyC** [comment]
Or you will, that's also an option :D

**Comment by vnesek** [comment]
What about npm users, is using yarn requirement now?

**Comment by FredyC** [comment]
No, these lifecycle scripts work the same across yarn & npm.

**Comment by vnesek** [comment]
Running on a system without yarn installed:

```
user@local> npm i

> mobx@5.14.1 postinstall /Users/vnesek/Workspace/ddex-front/node_modules/mobx
> yarn --cwd website install

sh: yarn: command not found

```

**Comment by FredyC** [comment]
Published 5.14.2 ... note there is no 4.14.2 as MobX 4 wasn't affected.

**Comment by benjamind** [comment]
@vnesek yarn has been required for mobx development for a long time now I think. It is *not* required to consume mobx as a dependency though.
